src/pages/home/home.ts

```diff
@@ -53,10 +54,14 @@ export class HomePage {
   //Random username.
   username: string;
 
+  // If central location is on water
+  onWater: boolean;
```

Comment:
**Best Practice**: Variables that are booleans should start with the prefix `is`.